dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/program/hibernate/Enrollment.hbm.xml

```diff
@@ -32,7 +32,7 @@
 
     <property name="enrollmentDate" column="enrollmentdate" not-null="true"/>
 
-    <property name="endDate" column="enddate"/>
+    <property name="completedDate" column="enddate"/>
```

Comment:
enrollmentDate is mapped to enrollmentdate column. Can we do the same for completedDate?